server/integration/testsuite/src/test/java/org/infinispan/server/test/cs/custom/CustomCacheStoreIT.java

```diff
@@ -78,49 +80,4 @@ public void test() throws Exception {
     private String getAttribute(MBeanServerConnectionProvider provider, String mbean, String attr) throws Exception {
         return provider.getConnection().getAttribute(new ObjectName(mbean), attr).toString();
     }
-
-    /*
-    * Credit goes to user577732 - http://stackoverflow.com/a/9305091/2064880 (slightly modified)
-    */
-    public static void addFilesToZip(File source, File[] files, String path){
-        try{
-            File tmpZip = new File(source.getAbsolutePath()+".tmp");
-            FileUtils.copyFile(source, tmpZip);
-
-            byte[] buffer = new byte[4096];
-            ZipInputStream zin = new ZipInputStream(new FileInputStream(tmpZip));
-            ZipOutputStream out = new ZipOutputStream(new FileOutputStream(source));
-            for(int i = 0; i < files.length; i++){
-                InputStream in = new FileInputStream(files[i]);
-                out.putNextEntry(new ZipEntry(path + files[i].getName()));
-                for(int read = in.read(buffer); read > -1; read = in.read(buffer)){
-                    out.write(buffer, 0, read);
-                }
-                out.closeEntry();
-                in.close();
-            }
-            for(ZipEntry ze = zin.getNextEntry(); ze != null; ze = zin.getNextEntry()){
-                if(!zipEntryMatch(ze.getName(), files, path)){
-                    out.putNextEntry(ze);
-                    for(int read = zin.read(buffer); read > -1; read = zin.read(buffer)){
-                        out.write(buffer, 0, read);
-                    }
-                    out.closeEntry();
-                }
-            }
-            out.close();
-            tmpZip.delete();
-        }catch(Exception e){
-            e.printStackTrace();
-        }
-    }
-
-   public static boolean zipEntryMatch(String zeName, File[] files, String path){
-        for(int i = 0; i < files.length; i++){
-            if((path + files[i].getName()).equals(zeName)){
-                return true;
-            }
-        }
-        return false;
-    }
 }
```

Comment:
Finally this hack could be removed! It really nice that server supports deployments now.